Contributing.md

```diff
@@ -0,0 +1,98 @@
+
+We also use [Gulp](https://gulpjs.com) for our build tool. Though Jekyll has support for Sass, we have opted to write plain CSS for 2019 browsers and let the tools backfill for compatibility. If you haven't worked with Gulp before, we recommend their [getting started docs](https://gulpjs.com/docs/en/getting-started/quick-start) which includes a link to [Node install steps](https://nodejs.org/en/)
+
+
+Clone the repository
+
+```{bash}
+git clone https://github.com/hurricane-response/response-theme.git
+```
+
+navigate to the directory
+
+```{bash}
+cd response-theme
+```
+
+install the Jekyll dependencies
+
+```{bash}
+bundle install
+```
+
+install the Gulp dependencies
+
+```{bash}
+npm i
+```
+
+`Gulpfile` has two main tasks: `build` and `watch`
+
+`build` does just that - builds the `_site` directory.
+
+`watch` watches both `assets/css/_inc` directory to check for CSS changes as well as Jekyll file changes and rebuilds the appopriate files.
+
+**A note about the CSS** All CSS is in the `_inc` directory is plain CSS. Current browser support for variables is strong, but we use PostCSS to fill in browser compatibility. PostCSS also compiles `assets/css/_inc/main.css` into `assets/css/main.css`. You should never need to edit `assets/css/main.css` file directly.
+
+Reminder that this is a [Jekyll theme](https://jekyllrb.com/docs/themes/), not a stand alone site.
+
+We optimized the build tools in this project with three goals:
+
+* quick deploy.
+* provide an out-of-the-box experience when using the theme as is.
+* allow overriding `main.css` in the individual response site for event specific customizations.
+
+## Where to start?
+
+The [issues](https://github.com/hurricane-response/response-theme/issues). We ask that you create an issue before starting work on something, even if a new feature. We also ask that you adhere as best as possible to the issue template to help us better triage. Certainly small,minor typos do not require a novel, but the more information the better.
+
+We have found that robust discussion before starting a new feature or large refactor has the greatest chance of success. Discussions can occur in the issue created or in Slack.
+
+
+**Note** sometimes during an active response focus is on immediate needs and those of our partners. We respectfully ask that you use the issues and `@` one of the contributors.
+
+### Whoa! I just want to fix a typo.
+
+Yes! We understand that frustration. GitHub does a great job of providing an in-browser experience for this scenario. If you don't have a GitHub account, first [create an account](https://github.com/join), then we ask you to create an issue for your suggested fix. Then navigate to the file with the minor change. In the upper right corner, next to the trash can symbol is a pencil. That icon will allow you to start the editing process. GitHub automagically will create a fork with your change and walk you through submitting it as a pull request.
```

Comment:
Is there a new-contributor-friendly documentation for this from github that we could link to?  It feels like we could link to a step-by-step tutorial, but I've not been able to suss out one with google searches.